### PR TITLE
Add MembersCanForkPrivateRepositories field to Organization struct

### PR DESCRIPTION
**Pull request by lmt-swallow**
This PR adds `MembersCanForkPrivateRepositories` field to `Organization` struct.

### CODE REVIEW

File: github/orgs.go [hunk 1]
```diff
@@ -65,6 +65,9 @@ type Organization struct {
 	MembersCanCreatePrivateRepos  *bool `json:"members_can_create_private_repositories,omitempty"`
 	MembersCanCreateInternalRepos *bool `json:"members_can_create_internal_repositories,omitempty"`
 
+	// MembersCanForkPrivateRepositories  toggles whether organization members can fork private organization repositories.
+	MembersCanForkPrivateRepositories *bool `json:"members_can_fork_private_repositories,omitempty"`
```
Comment (gmlewis) [inline github/orgs.go:69]:
```suggestion
	MembersCanForkPrivateRepos *bool `json:"members_can_fork_private_repositories,omitempty"`
```

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thank you, @lmt-swallow!
Let's please shorten the field name like it's neighbors.

File: github/orgs.go (hunk 1 above)
Comment (lmt-swallow) [reply github/orgs.go]: Thank you so much! I've shortend the name and run `go generate/test/vet` at https://github.com/google/go-github/pull/2389/commits/b510ad5fd6341045d97c761c38b07c337905d0d4.

**Review by gmlewis — APPROVED** [review]
Thank you, @lmt-swallow !
LGTM.
Merging.

**Pull request merged**